### DISCUSSION TITLE
500 Internal Server Error

### DISCUSSION THREAD

**Discussion by Draku3l** [deeztek/Hermes-Secure-Email-Gateway#28] [Support] [answered]
Hi,

i installed a fresh Ubuntu 18.04.6 on a Hyper V machine, ran the installation of Hermes Secure Email Gateway and now that it is installed, I can't get to the admin console..
 When I goto https://myipaddress/admin I get 500 Internal Server Error (nginx/1.14.0 (Ubuntu)).

Please advise?

**Comment by deeztek** [comment, marked as answer]
That's normal behavior until the **Authelia authentication and authorization server** service starts. If you refresh your web browser, it should come up with the **Sign In** Page.

**Reply by deeztek** [reply]
Please ensure you take a look at the [Getting Started](https://docs.deeztek.com/books/hermes-seg-administrator-guide/page/getting-started) Guide.  This error is addressed in there also.

**Reply by deeztek** [reply]
@Draku3l are you still having an issue or can I close this question as answered?

**Reply by Draku3l** [reply]
I will only be able to check when I get back into the office tomorrow, as the server is powered off for now.

**Reply by Draku3l** [reply]
Still no luck..

I find that when the server is powered on hermesseg service fails to start, which I then force it to start with admin credentials. 15 minutes later I am still get no joy  with the internal server error.

hermes error log "2021/12/11 09:02:35 [error] 1569#1569: *25 connect() failed (111: Connection refused) while connecting to upstream, client: myip, server: , request: "GET /favicon.ico HTTP/1.1", upstream: "http://127.0.0.1:9091/favicon.ico", host: "serverip", referrer: "https://spam1.integratedit.co.za/admin/"

**Reply by deeztek** [reply]
Hi @Draku3l the error you posted is not the hermesseg service. That's the Authelia service.  Please join me on our support Matrix channel at https://matrix.to/#/#hermesseg:matrix.org to further troubleshoot this issue.

Thanks
